src/ngraph/runtime/intelgpu/intelgpu_op_custom_kernels.hpp

```diff
@@ -142,6 +152,12 @@ namespace ngraph
                               const std::vector<Shape>& input_shapes,
                               const std::string& output_type,
                               const Shape& output_shape);
+            void gen_window_loop(codegen::CodeWriter& writer,
```

Comment:
Forgot to mention it before. Do we really need this function exposed externally? I think it can be static?